plotsearch/templates/mixins/section.html

```diff
@@ -1,42 +1,41 @@
 {% load entry_filter %}
+{% load form_visible %}
 
 {% if section.fields.all and section.visible == True %}
-<div>
-    <h2>{{ section }}</h2>
-    <table>
-        {% for field in section.fields.all %}
-            {% if field.enabled == True%}
-                <tr>
-                    <th>
-                        {{field}}
-                    </th>
+    <div>
+        <h2>{{ section }}</h2>
+        <table>
+            {% for field in section.fields.all %}
+                {% if field.enabled == True %}
+                    <tr>
+                        <th>
+                            {{ field }}
+                        </th>
                         {% for entry in field.entry_set.all|filter_answer:answer %}
-                        <td>
-                            {% if field.identifier == "hakija" %}
-                                {% if entry.value == "0" %}
-                                    Henkilö
-                                {% endif %}
-                                {% if entry.value == "1" %}
-                                    Yritys
+                            <td>
+                                {% if field.type.identifier == "checkbox" or field.type.identifier == "dropdown" or field.type.identifier == "radiobutton" or field.type.identifier == "radiobuttuninline" %}
```

Comment:
typo: radiobuttuninline should be radiobuttoninline

Also if field.type.identifier == "dropdown" throws ValueError